pkg/apis/hive/v1/hiveconfig_types.go

```diff
@@ -57,6 +57,10 @@ type HiveConfigSpec struct {
 
 	// DeprovisionsDisabled can be set to true to block deprovision jobs from running.
 	DeprovisionsDisabled *bool `json:"deprovisionsDisabled,omitempty"`
+
+	// CLIImage can be set to override the location of the image with
+	// the oc command line tool.
+	CLIImage string `json:"cliImage,omitempty"`
```

Comment:
I am a little concerned about the name of this variable. The CLI image term is overloaded in Hive, where it could mean the CLI image used to extract images from the release image or the CLI image used by the installmanager to run oc adm commands. I think we should be a bit more specific with the name of this variable here since this is exposed directly to the end user. How do you feel about `ImageExtractionCLIImage`? It is a bit wordy.

Reply:
+1, that name sounds ok to me.

Change looks great in general, thanks for wiring this up @dhellmann

Reply:
I've updated the name.